src/components/molecules/NavSearchBar/NavSearchResult.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import classNames from "classnames";
+
+import "./NavSearchResult.scss";
+
+interface NavSearchResultProps {
+  title: string;
+  description?: string;
+  image?: string;
+  isAvatar?: boolean;
+  onClick?: (e?: React.MouseEvent) => void;
+}
+
+export const NavSearchResult: React.FC<NavSearchResultProps> = ({
+  title,
+  description,
+  image,
+  isAvatar,
+  onClick,
+}) => {
+  return (
+    <div className="nav-dropdown--search-result" onClick={onClick}>
+      <div
+        className={classNames("nav-dropdown--search-result-image", {
+          "nav-dropdown--search-result-avatar": isAvatar,
+        })}
+        style={{
+          backgroundImage: image ? `url(${image})` : "",
+        }}
+      />
+      <div className="nav-dropdown--search-result-title">
```

Comment:
I don't think I saw the styles for this in `src/components/molecules/NavSearchBar/NavSearchResult.scss`.

Is it defined somewhere else? Should probably be moved into this file + renamed for BEM.

Reply:
yeah, you're right. :see_no_evil: haven't noticed. Thanks!
fixed: https://github.com/sparkletown/sparkle/pull/1328/commits/eb24607d724c23e94c3d4d1a40d7df37f6d9489b

Reply:
Ha, no worries :)